src/components/user/uc/registrationPin.uc.js

```diff
@@ -8,7 +8,12 @@ const deleteRegistrationPins = async (pinIds) => {
 	return registrationPinRepo.deleteRegistrationPins(pinIds);
 };
 
+const deleteRegistrationPinsByEmail = async (email) => {
+	return { complete: true, data: registrationPinRepo.deleteRegistrationPinsByEmail(email) };
+};
+
 module.exports = {
 	getRegistrationPinsForUser,
```

Comment:
this is a UC, i think we need permissions check for it. 
deleteRegistrationPinsByEmail

Reply:
Permission Check is done by the service. Not needed again for the use case